protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1169,7 +1169,7 @@ message OracleDataConnection {
 
 // -- Markets --
 message GetMarketRequest {
-    string market_id = 1;
+    string id = 1;
```

Comment:
Did you mean to leave this rename in?

Reply:
No, I meant to make it consistent with asset, node, proposal, party - in their own objects they are all queried by `Id`, not by `ObjectId`